lib/scarpe/display_service.rb

```diff
@@ -181,4 +184,74 @@ def query_display_widget_for(id, nil_ok: false)
       display_widget
     end
   end
+
+  class GlimmerDisplayService
```

Comment:
I'm gonna extract the WebviewDisplayService into another file in the webview directory. But for now, this is great.

Reply:
Also, a lot of this is still doing it the Webview way (e.g. it says to create WebviewApp and it's looking up class names for Webview widgets.) So I'm assuming this is just an initial copy-paste? In the final version you'll want to look up the Glimmer stuff, no Webview.